packages/host/src/CommandK.tsx

```diff
@@ -166,13 +166,6 @@ export const CommandK: FC = ({ children }) => {
             .build()
         ),
     },
-    {
```

Comment:
offroad: found that while I was testing on the demo server I was accidentally mashing the `r` key and getting a reports placeholder page 🙄